Naming.md

```diff
@@ -46,3 +53,54 @@ protocol FooEventHandling {
     // good
 }
 ```
+
+### Naming a delegate method
+
+The first parameter should always be the item that the delegate was called for.
+
+#### Example:
+
+```swift
+// really bad: there's no parameter for the item that the delegate was called for
```

Comment:
I'm not convinced that this is the best "bad" example since it seems that `draft` is what this delegate is called for.